spec/lib/miq_automation_engine/miq_ae_service_rbac_spec.rb

```diff
@@ -0,0 +1,213 @@
+include AutomationSpecHelper
+module MiqAeServiceModelSpec
+  include MiqAeEngine
+  describe MiqAeMethodService::MiqAeServiceVmOrTemplate do
+    before do
+      vm11
+      vm21
+      user1
+      user2
+    end
+
+    after do
+      Thread.current.thread_variable_set(:miq_rbac, false)
+    end
+
+    let(:default_tenant) { Tenant.seed }
+    let(:enable_rbac) { Thread.current.thread_variable_set(:miq_rbac, true) }
```

Comment:
use a function for this

this is setting a global variable and not cleaning up after itself. this will cause sporadic test failures